tls-tester/pom.xml

```diff
@@ -15,11 +15,11 @@
       <plugin>
         <groupId>com.axonivy.ivy.ci</groupId>
         <artifactId>project-build-plugin</artifactId>
-        <version>8.0.6</version>
+        <version>11.2.0</version>
+        <extensions>true</extensions>
         <configuration>
-        	<ivyVersion>[8.0.15,9.1.0)</ivyVersion>
+          <ivyVersion>[8.0.15,9.1.0)</ivyVersion>
```

Comment:
I think you should remove that <ivyVersion> tag and its content ... otherwise you're building an 11.2 project with a 8.x engine